wait_for.sh

```diff
@@ -10,7 +10,7 @@ TOP_PID=$$
 KUBECTL_ARGS=""
 WAIT_TIME="${WAIT_TIME:-2}" # seconds
 DEBUG="${DEBUG:-0}"
-TREAT_ERRORS_AS_READY=0
+TREAT_ERRORS_AS_READY="${TREAT_ERRORS_AS_READY:-0}"
```

Comment:
I would love for this to be a variable that we could control, but unfortunately I think the `if/elif` block starting at line 171 will overwrite whatever the user tries to supply. I might recommend that we make the default `-1` and only set it based on the resource type when it's not `-1`.

Reply:
Wait a minute, I think I just realized how this is meant to work. You can use the resource type ending in `-wr`, probably for "with restart" or something like that, to achieve the behavior I'm after. It might work for you too.

Reply:
thanks - I missed that!